dist/profile/manifests/jiraveg.pp

```diff
@@ -0,0 +1,32 @@
+# profile for jira
+class profile::jiraveg (
+  $jira_javahome,
+  $jira_version
+){
+  $pkgs = ['openjdk-7-jre-headless', 'facter']
```

Comment:
facter is installed on all our machines, but that doesn't necessarily mean that `/etc/facter` exists (it doesn't)